README.md

```diff
@@ -4,3 +4,10 @@
 [![Coverage](https://codecov.io/gh/devmotion/RealDot.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/devmotion/RealDot.jl)
 [![Coverage](https://coveralls.io/repos/github/devmotion/RealDot.jl/badge.svg?branch=main)](https://coveralls.io/github/devmotion/RealDot.jl?branch=main)
 [![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
+
+This package only contains and exports a single function `realdot(x, y)`. It computes
+`real(LinearAlgebra.dot(x, y))` while avoiding computing the imaginary part of
+`LinearAlgebra.dot((x, y)` if possible.
+
```

Comment:
I would add a bit here about what this means, or why this is generally useful. e.g.

```suggestion
The real dot product is useful when one treats complex numbers as embedded in a real vector space.
For example, take the complex arrays ``x = a + i b`` and ``y = c + i d``. Their real dot product is
`real(dot(x, y)) == dot(real(x), real(y)) + dot(imag(x), imag(y))`. This is the same result one would get by reinterpreting the arrays as real arrays:
```julia
xreal = reinterpret(real(eltype(x)), x)
yreal = reinterpret(real(eltype(y)), y)
real(dot(x, y)) == dot(xreal, yreal)
```
```